src/Server/RestRouterHandlers/IngestStatusHandler.h

```diff
@@ -18,7 +18,7 @@ class IngestStatusHandler final : public RestRouterHandler
 
 private:
     bool validatePost(const Poco::JSON::Object::Ptr & payload, String & error_msg) const override;
-    bool streaming() const override { return false; }
+    bool streamingInput() const override { return false; }
```

Comment:
Nit : i think we can remove this overridden function as the default implementation is `false`. Making it explicit here doesn't add readability but have this kind of maintenance overhead  like renaming